scripts/env/cd.sh

```diff
@@ -158,13 +158,13 @@ cd() {
 
         [[ "${GVM_DEBUG}" -eq 1 ]] && echo "Switching to: ${use_goversion}"
 
-        \gvm use --quiet "${use_goversion}" || return 1
+        source \gvm use --quiet "${use_goversion}" || rslt=1
```

Comment:
In general, `gvm` should be in the user's path at this point as it is sourced in `$GVM_ROOT/scripts/gvm-default` via `"$HOME/.gvm/scripts/gvm"`.

Reply:
Also, the design pattern in here is to return immediately with an error condition. The `$rslt` variable is used to capture and preserve return values from methods that conform to the RETVAL convention.